test_fms/fms2_io/Makefile.am

```diff
@@ -1,33 +1,40 @@
-# This is an automake file for the test_fms/diag_manager directory of the FMS
+# This is an automake file for the test_fms/fms2_io directory of the FMS
 # package.
 
-LOG_DRIVER = env AM_TAP_AWK='$(AWK)' $(SHELL) \
-             $(top_srcdir)/tap-driver.sh
+# J.Liptak, R.Menzel
 
 # Find the fms_mod.mod file.
-AM_CPPFLAGS = -I${top_builddir}/mpp
-AM_CPPFLAGS += -I${top_builddir}/fms2_io
+AM_CPPFLAGS = -I${top_builddir}/mpp -I${top_builddir}/fms2_io
+
+# Link to the FMS library.
+LDADD = ${top_builddir}/libFMS/libFMS.la
 
 # Build this test program.
-check_PROGRAMS = test_atmosphere_io
+check_PROGRAMS = test_fms2_io test_atmosphere_io
 
 # This is the source code for the test.
-test_atmosphere_io_SOURCES = argparse.F90 setup.F90 test_atmosphere_io.F90
-test_atmosphere_io_LDADD = ${top_builddir}/libFMS/libFMS.la
-EXTRA_DIST = test_atmosphere_io.sh test_atmosphere_io.bats input.nml
+test_fms2_io_SOURCES = argparse.F90 test_fms2_io.F90 create_atmosphere_domain.inc \
+                       create_land_domain.inc create_ocean_domain.inc \
+                       ocean_restart_file_test.inc land_compressed_restart_file_test.inc \
+                       land_unstructured_restart_file_test.inc
+test_atmosphere_io_SOURCES = argparse.F90 setup.F90 test_atmosphere_io.F90 \ 
```

Comment:
This line (20) has an extra space at the end of the line, which causes autoreconf -i to issue this warning:

`test_fms/fms2_io/Makefile.am:20: warning: whitespace following trailing backslash`